json-webhook/functions/event-to-webhook.js

```diff
@@ -0,0 +1,29 @@
+const fetch = require('node-fetch');
+
+exports.handler = async function(context, event, callback) {
+  try {
+    // IFTTT only pulls the fields value1, value2, and value3 from webhook JSON;
+    event.value1 = event.MessageSid;
+    event.value2 = event.From;
+    event.value3 = event.Body;
+
+    const res = await fetch(context.WEBHOOK_URL, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(event),
+    });
+
+    if (res.ok) {
+      callback(null, {});
+    }
+    else {
+      console.log(res.statusText);
```

Comment:
When you respond with an error in the first argument to `callback` then that will be logged and exposed in the debugger. So we shouldn't need to log anything here.

Further, if you ever need to log an error that isn't also passed on to `callback`, using `console.error` or `console.warn` will also fire a debugger event. [Logs also store their error level and you can use `info`, `warn` or `error`](https://www.twilio.com/docs/runtime/functions-assets-api/api/logs).